### DISCUSSION TITLE
Can't detect collisions events without Dynamic RigidBody

### DISCUSSION THREAD

**Discussion by Gugustinette** [dimforge/rapier#724] [Q&A]
Hi !

So basically, I'm creating a game engine (in TypeScript) based on Rapier and rapier.js.

I created a collision system based on [advanced collision detection](https://rapier.rs/docs/user_guides/javascript/advanced_collision_detection_js#collision-and-contact-force-events), which works great but only with colliders that are attached to a dynamic rigidbody, and I don't know why 😭
From what I've understand of Rapier, the collisions should work between any collider what so ever.

Basically, my scene is :
- A ground that has a collider alone
- A character that has a collider attached to a KinematicPositionBased rigidbody (+ a collider sensor that is the one to detect events)
- A cube that has a collider attached to a Dynamic rigidbody

So I get collisions between the character and the cube, but not between the character and the ground. 😕

**Comment by Gugustinette** [comment]
Alright something I probably couldn't understand by myself is that I needed a rigidbody to be attached to my collider so the event can be emitted.

Attaching a KinematicPositionBased did the trick. Maybe this wasn't very clear in the documentation ?
